sdk/containerregistry/container-registry/samples/javascript/containerRegistryClient.js

```diff
@@ -13,11 +13,33 @@ async function main() {
   const endpoint = process.env.ENDPOINT || "<endpoint>";
 
   const client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
+  await listRepositories(client);
+  // await deleteRepository(client);
```

Comment:
Is commenting out deleteRepository intentional?

Reply:
Yes (I will add some comments above it)

Reply:
Will uncomment instead.